crates/wallet/src/wallet/tx_builder.rs

```diff
@@ -758,14 +763,12 @@ pub enum TxOrdering {
 
 impl TxOrdering {
     /// Sort transaction inputs and outputs by [`TxOrdering`] variant
-    pub fn sort_tx(&self, tx: &mut Transaction) {
+    pub fn sort_tx(self, tx: &mut Transaction, rng: &mut impl RngCore) {
```

Comment:
❓ For consistency shouldn't this be called `sort_tx_with_aux_rand`? Then in the follow-up PR that adds the `rand` feature flag we can cfg back in a `sort_tx` without the `rng` param?

Reply:
Makes sense, will amend soon.

Reply:
Renamed